x/stakeibc/keeper/icqcallbacks_callibrate_delegation.go

```diff
@@ -4,13 +4,15 @@ import (
 	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
-	"github.com/cosmos/gogoproto/proto"
 
 	"github.com/Stride-Labs/stride/v16/utils"
 	icqtypes "github.com/Stride-Labs/stride/v16/x/interchainquery/types"
 	"github.com/Stride-Labs/stride/v16/x/stakeibc/types"
 )
 
+// CalibrationThreshold is the max amount of tokens by which a calibration can alter internal record keeping of delegations
+var CalibrationThreshold = sdk.NewInt(5000)
```

Comment:
yeah I would have assumed that as well. 

It's totally it's possible that it's caused by something else on their chain, but unfortunately it's closed source :(

Reply:
my understanding from other chains is that's it's only due to the rounding error - cause on other chains, it only differs by a couple utokens

Reply:
thinking about this a little more - I've noticed that often when a validator is slashed, it seems to happen in two increments (i.e. the shares_to_tokens_rate changes twice)

Very possible that's what happened here and we would have been able to resolve it with a normal slash tx (instead of the calibration query). 

Either way, I don't have any concern with 5000 and we can lower it later if get more intel on injective!